src/lib.rs

```diff
@@ -10,6 +10,7 @@ extern crate memmap;
 #[cfg(test)]
 extern crate quickcheck;
 extern crate rand;
+extern crate rand_distr;
 #[cfg(test)]
 extern crate tempdir;
 extern crate time;
```

Comment:
There is no longer any need to specify `extern crate` in modern Rust. The things that are configured with `#[cfg(test)]` can be put into dev-dependencies instead.

Reply:
I understand that it is a bit unrelated, so it can easily be fixed in a different PR.

Reply:
Alright, I can clean all those `extern crates` in a next PR to modernize everything :+1: